resources/user-agents/browsers/yabrowser/yabrowser-1-20-14-12.json

```diff
@@ -269,6 +269,68 @@
           ]
         }
       ]
+    },
+    {
+      "userAgent": "Yandex Browser #MAJORVER#.#MINORVER# for iOS",
```

Comment:
Did find a different URL that archive.org does have indexed:

https://web.archive.org/web/20150425230024/https://itunes.apple.com/us/app/yandex.browser-for-iphone/id483693909?mt=8

But still only goes back to 15.4 (April 25, 2015).